desc/magnetic_fields/_core.py

```diff
@@ -451,6 +510,22 @@ def save_mgrid(
         B_phi = field[:, 1].reshape(nphi, nZ, nR)
         B_Z = field[:, 2].reshape(nphi, nZ, nR)
 
+        # evaluate magnetic vector potential on grid
+        try:
```

Comment:
might be better to make saving the vector potential a user option (so that we can reduce file size and compute time if its not needed), default to True, and throw an actual error if it fails